src/System.Text.Json/src/System/Text/Json/Document/JsonDocument.cs

```diff
@@ -75,6 +75,29 @@ public void Dispose()
             }
         }
 
+        /// <summary>
+        ///  Write the document into the provided writer as a JSON value.
+        /// </summary>
+        /// <param name="writer"></param>
+        /// <exception cref="ArgumentNullException">
+        ///   Parameter <paramref name="writer"/> of type <see cref="Utf8JsonWriter"/> should not be null.
```

Comment:
The standard boilerplate for this is more like

```suggestion
        ///   The <paramref name="writer"/> parameter is <see langword="null"/>.
```

(It describes when the exception is thrown, vs the guidance to avoid it)

Reply:
corrected it.